djangocms_versioning/cms_config.py

```diff
@@ -358,7 +357,7 @@ def change_innavigation(self, request, object_id):
         try:
             version.check_modify(request.user)
         except ConditionFailed as e:
-            self.message_user(request, force_str(e), messages.ERROR)
+            # Send error message
```

Comment:
What's the change here - don't message the user because they're getting a 403 response anyway?

Reply:
Indeed. The 403 response triggers a clear message. Sending another one leads to the 403 message being shown - letter for letter the same as the previously sent - once more out of context with the next request response cycle.